php7x packages need new binaries

Due to `libressl` update, `php7*` packages have to be rebuilt. This error is thrown : 
```shell
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: warning: libssl.so.1.0.0, needed by /usr/local/lib64/libsybdb.so, not found (try using -rpath or -rpath-link)
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: warning: libssl.so.1.0.0, needed by /usr/local/lib64/libsybdb.so, not found (try using -rpath or -rpath-link)
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: warning: libcrypto.so.1.0.0, needed by /usr/local/lib64/libsybdb.so, not found (try using -rpath or -rpath-link)
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: warning: libcrypto.so.1.0.0, needed by /usr/local/lib64/libsybdb.so, not found (try using -rpath or -rpath-link)
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: warning: libssl.so.47, needed by /usr/local/lib64/libldap.so, not found (try using -rpath or -rpath-link)
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: warning: libcrypto.so.45, needed by /usr/local/lib64/libldap.so, not found (try using -rpath or -rpath-link)
: warning: libssl.so.47, needed by /usr/local/lib64/libldap.so, not found (try using -rpath or -rpath-link)
/usr/local/lib64/gcc/x86_64-cros-linux-gnu/10.2.0/../../../../x86_64-cros-linux-gnu/bin/ld: warning: libcrypto.so.45, needed by /usr/local/lib64/libldap.so, not found (try using -rpath or -rpath-link)
/bin/bash /usr/local/tmp/crew/php-7.4.11.tar.xz.dir/php-7.4.11/libtool --silent --preserve-dup-deps --mode=install cp ext/opcache/opcache.la /usr/local/tmp/crew/php-7.4.11.tar.xz.dir/php-7.4.11/modules
Generating phar.php
/usr/local/tmp/crew/php-7.4.11.tar.xz.dir/php-7.4.11/sapi/cli/php: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
make: *** [Makefile:473: ext/phar/phar.php] Error 127
make: *** Waiting for unfinished jobs....
php74 failed to install: `make V=0 -j2` exited with 2
```
`libssl.so.1.0.0` is still missing. I am investigating deeper to see what else shall be rebuilt

--- a/packages/freetds.rb
+++ b/packages/freetds.rb
@@ -3,22 +3,14 @@ require 'package'
 class Freetds < Package
   description 'FreeTDS is a set of libraries for Unix and Linux that allows your programs to natively talk to Microsoft SQL Server and Sybase databases.'
   homepage 'http://www.freetds.org/'
-  version '1.00.109'
+  version '1.2.9'
   compatibility 'all'
-  source_url 'ftp://ftp.freetds.org/pub/freetds/stable/freetds-1.00.109.tar.bz2'
-  source_sha256 '3e93b2bcdddc7246147398b9bc3b989c6e7ffed54acbce18f4f34b745c8f0034'
+  source_url 'ftp://ftp.freetds.org/pub/freetds/stable/freetds-1.2.9.tar.bz2'
+  source_sha256 '90d7c2553d86fcca0029f118a2d61b48d69eed193549c4ff7306c5f8e132c2f5'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/freetds-1.00.109-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/freetds-1.00.109-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/freetds-1.00.109-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/freetds-1.00.109-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'e2ee2213f781c3f36180e01cf9c5123fdab8d3528172ceb25b19b53a84210248',
-     armv7l: 'e2ee2213f781c3f36180e01cf9c5123fdab8d3528172ceb25b19b53a84210248',
-       i686: '34473f6c9cf9512da0de7017a7522d43a60f6ea8a78672269166a85e55dc3353',
-     x86_64: 'cda228331626e14d559eb680d7a157d1545e24d60f3b5db49d1fbb54180206bd',
   })
 
   depends_on 'unixodbc'

